RevPiDevice.c

```diff
@@ -139,12 +139,6 @@ void RevPiDevice_init(void)
 	RevPiDevice_incDevCnt();
 }
 
```

Comment:
Should not this be handled in a seperate ticket, since it is an unrelated bugfix?

Reply:
Strictly speaking, yes. But from my POV it would just mean additional red tape without benefit. I did make sure to use separate commits, so the git history is clean.